src/pages/PageOrgchart.vue

```diff
@@ -196,7 +197,7 @@ export default {
         const orgchart = await data.json();
         this.tree = orgchart.forrest;
         this.loose = orgchart.loose;
-        if (this.$route.name !== 'OrgchartHighlight') {
+        if (!this.orgFromStore && this.$route.name !== 'OrgchartHighlight') {
```

Comment:
```suggestion
        if (this.$route.name === 'OrgchartHighlight') {
          this.collapseAll();
        } else if (!this.orgFromStore) {
```
I would do collapse first when highlighting